tools/lightning/csv.md

```diff
@@ -179,14 +179,13 @@ trim-last-separator = false
 示例内容：
 
 ```
-ID	Region	Count
-1	East	32
-2	South	NULL
-3	West	10
-4	North	39
+ID Region Count
```

Comment:
- For this code block, we can directly copy that in [the English file](https://github.com/pingcap/docs/blob/master/tools/lightning/csv.md).
- Generally, we should keep the original length of spaces in a code block.

Please use the following version:

```
ID    Region    Count
1     East      32
2     South     NULL
3     West      10
4     North     39
```